media/css/base/page-not-found.scss

```diff
@@ -2,79 +2,68 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-@import '../pebbles/includes/lib';
-@import '../hubs/sections';
+@import '../../protocol/css/includes/lib';
 
-.mzp-c-navigation + #outer-wrapper,
-.moz-global-nav-drawer + #outer-wrapper {
-    padding-top: 40px;
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+.no-highlight:link, .no-highlight:visited, .no-highlight:hover, .no-highlight:active {
+    color: black;
 }
 
 main {
-    min-height: 450px;
-
-    header {
-        margin-bottom: 40px;
+    font-size: 18px;
```

Comment:
Nit: try and avoid setting aritrary font sizes in pixels, and stick to the predefined type mixins defined in protocol? https://protocol.mozilla.org/fundamentals/typography.html#styles

Same applies below

Reply:
I originally had used the mix-ins but was asked by design not to and set the font sizes manually with specific break points for this page. (https://github.com/mozilla/bedrock/issues/8407#issuecomment-587989081) I reached out to Cristina and asked. The mixins we have defined don't match the sizing designed for the page.

Reply:
We spent a long time trying to standardise on type-scale across mozorg pages, which at one time lacked any real kind of consistency. I can only encourage you to encourage our designers to try and use what’s in the design system where possible. If we really can’t avoid setting arbitrary type sizes, then I’d suggest using the `font-size()` mixin, so modern browsers will get rem units instead of fixed pixel sizes.